### PR TITLE
Add overheatInterval and remove pressure

### PR DESCRIPTION
**Pull request by dacolgit**
## Purpose

* added overheatInterval and removed pressure

## Does this introduce a breaking change?

```
[ ] Yes
[x] No
```

## Pull Request Type
What kind of change does this Pull Request introduce?

```
[ ] Bugfix
[x] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Documentation content changes
[ ] Other... Please describe:
```

## How to Test
*  Get the code

```
git clone [repo-address]
cd [repo-name]
git checkout [branch-name]
npm install
```

* Test the code

```
```

## What to Check
Verify that the following are valid
* ...

### CODE REVIEW

File: src/PluginNodes/Boiler2PluginNodes.cs [hunk 1]
```diff
@@ -58,6 +59,11 @@ public void AddOptions(Mono.Options.OptionSet optionSet)
             $"Boiler #2 required maintenance interval in minutes\nDefault: {_maintenanceIntervalMinutes}",
```
Comment (luiscantero) [inline src/PluginNodes/Boiler2PluginNodes.cs:59]: Change the description to seconds and rename the variable as well

File: src/PluginNodes/Boiler2PluginNodes.cs [hunk 2]
```diff
@@ -25,7 +25,7 @@ public class Boiler2PluginNodes : IPluginNodes
     private BaseDataVariableState _maintenanceIntervalMinutesNode;
     private BaseDataVariableState _overheatThresholdDegreesNode;
     private BaseDataVariableState _currentTempDegreesNode;
-    private BaseDataVariableState _pressureNode;
+    private BaseDataVariableState _overheatIntervalNode;
```
Comment (luiscantero) [inline src/PluginNodes/Boiler2PluginNodes.cs:28]: Let's call this _overheatIntervalSecondsNode to be consistent with the others. Also, pls move to be below the maintenance interval

File: src/PluginNodes/Boiler2PluginNodes.cs [hunk 3]
```diff
@@ -58,6 +59,11 @@ public void AddOptions(Mono.Options.OptionSet optionSet)
             $"Boiler #2 required maintenance interval in minutes\nDefault: {_maintenanceIntervalMinutes}",
             (string s) => _maintenanceIntervalMinutes = (uint)CliHelper.ParseInt(s, min: 1, max: int.MaxValue, optionName: "boiler2maintint"));
 
+        optionSet.Add(
+            "b2oi|boiler2overheatinterval=",
+            $"Boiler #2 simulation overheat interval in seconds\nDefault: {_overheatIntervalSeconds}",
```
Comment (luiscantero) [inline src/PluginNodes/Boiler2PluginNodes.cs:64]: Pls remove simulation from the text

**Review by luiscantero — CHANGES_REQUESTED** [review]

File: src/PluginNodes/Boiler2PluginNodes.cs (hunk 1 above)
Comment (dacolgit) [reply src/PluginNodes/Boiler2PluginNodes.cs:59]: changed

File: src/PluginNodes/Boiler2PluginNodes.cs (hunk 3 above)
Comment (dacolgit) [reply src/PluginNodes/Boiler2PluginNodes.cs:64]: removed

**Review by luiscantero — APPROVED** [review]
:shipit:

**Pull request merged**